### PR TITLE
Use Stale bot to close issues

### PR DESCRIPTION
**Pull request by autoantwort**
I don't know if we want to do that in general but maybe it can safes resources to do more meaningful tasks.

### CODE REVIEW

**Comment by Cheney-W** [comment]
@vicroms  Could you please help review this PR? Thank you!

**Comment by vicroms** [comment]
In general, we like this change we are discussing internally about timelines for automatically closing PRs and issues. I'll have an update on this by next Tuesday.

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: 'We hope the question was answered satisfactorily. This issue will be automatically closed in 7 days if there is no response.'
+          only-labels: 'category:question'
+          days-before-issue-stale: 10
+          days-before-pr-stale: -1
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: 'We could not reproduce this error. Does the problem still exists? This issue will be automatically closed in 7 days if there is no response.'
```
Comment (Hoikas) [inline .github/workflows/stale.yml:18]:
```suggestion
          stale-issue-message: 'We could not reproduce this error. Does the problem still exist? This issue will be automatically closed in 7 days if there is no response.'
```

**Comment by Thomas1664** [comment]
Maybe it's worth adding `requires:more-information` as well?

**Comment by Cheney-W** [comment]
@BillyONeal Could you please take a look this PR?

**Comment by BillyONeal** [comment]
@AugP Did you want to tweak any of the figures in here?

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          days-before-pr-stale: -1
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: 'We could not reproduce this error. Does the problem still exist? This issue will be automatically closed in 7 days if there is no response.'
+          only-labels: 'requires:repro'
+          days-before-issue-stale: 60
+          days-before-pr-stale: -1
+      - uses: actions/stale@v6
+        with:
+          close-issue-message: "If you wish to add this port in the future, you can reopen this issue, but we're closing it for now."
```
Comment (quyykk) [inline .github/workflows/stale.yml:24]: Why close new port requests?

Comment (BillyONeal) [reply .github/workflows/stale.yml:24]: If it's been outstanding for 2+ months with no activity it suggests that the issue being present isn't going to change anything, whether it's a new port request or no.

**Comment by autoantwort** [comment]
@BillyONeal @AugP Any news on this topic?

**Comment by JonLiu1993** [comment]
@JavierMatosD, Could you please help take a review? Thanks.

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: 'We hope the question was answered satisfactorily. This issue will be automatically closed in 7 days if there is no response.'
```
Comment (dan-shaw) [inline .github/workflows/stale.yml:12]: From vcpkg team: Can you replace the issue message with `This is an automated message. Per our repo policy, stale issues get closed if there has been no activity in the past 28 days. The issue will be automatically closed in 14 days. If you wish to keep this issue open, please add a new comment.`

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+          only-labels: 'category:question'
+          days-before-issue-stale: 10
+          days-before-pr-stale: -1
+      - uses: actions/stale@v6
+        with:
+          stale-issue-message: 'We could not reproduce this error. Does the problem still exist? This issue will be automatically closed in 7 days if there is no response.'
```
Comment (dan-shaw) [inline .github/workflows/stale.yml:18]: Same here

File: .github/workflows/stale.yml (hunk 2 above)
Comment (dan-shaw) [inline .github/workflows/stale.yml:24]: Same format here

**Review by dan-shaw — COMMENTED** [review]
Sorry for the late reply! From vcpkg team: For stale issues, we should warn users at 14 days, and close the issue at 28 days. For new port requests, I think we can close the issue at 60 days, with a notice 14 days prior.

File: .github/workflows/stale.yml (hunk 3 above)
Comment (autoantwort) [reply .github/workflows/stale.yml]: I have updated the message

**Review by dan-shaw — APPROVED** [review]

**Comment by JackBoosY** [comment]
Thx for this!

**Pull request merged**